includes/class-wcpdf-settings-upgrade.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace WPO\WC\PDF_Invoices;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( !class_exists( '\\WPO\\WC\\PDF_Invoices\\Settings_Upgrade' ) ) :
+
+class Settings_Upgrade {
+
+	function __construct()	{
+		add_action( 'wpo_wcpdf_after_settings_page', array( $this, 'extension_overview' ), 10, 2 );
+	}
+
+	public function extension_overview( $tab, $section ) {
+		if ( $tab === 'upgrade' ) {
+			
+			$features = array(
+				array( 
+					'label' => __( 'Proforma invoice & Credit note', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description' => __( 'Update your workflow and handle refunds.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'extensions' => ['pro', 'bundle'],
+				),
+				array( 
+					'label' => __( 'Dropbox / FTP upload', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I would call it `Cloud storage upload`, because we plan to also add a few more like GDrive in the future. Maybe you could add the Dropbox and FTP in the description below? I also would mention **SFTP**.

Reply:
Yes, great suggestions. I will add those and change the cloud storage label.